app/containers/MessageBox/forceJpgExtension.ts

```diff
@@ -0,0 +1,12 @@
+import { ImageOrVideo } from 'react-native-image-crop-picker';
+
+import { isIOS } from '../../utils/deviceInfo';
+
+const regex = new RegExp(/\.[^/.]+$/); // Check from last '.' of the string
+
+export const forceJpgExtension = (attachment: ImageOrVideo) => {
```

Comment:
It's just a detail, but put the function return, as it will return the parameter informed, just use the same for the return.
`export const forceJpgExtension = (attachment: ImageOrVideo): ImageOrVideo => {`